assets/src/css/components/_menu--header.scss

```diff
@@ -0,0 +1,169 @@
+//
+// Menu in the header.
+//
+
+.menu {
+
+	&--header {
+		display: flex;
+		flex-flow: column nowrap;
+		margin: 0;
+		padding: 0;
+		list-style: none;
+		font-size: 24px;
+
+		@media ( query( min, tablet ) ) {
+			margin: 0 -10px;
+			flex-flow: row nowrap;
+			font-size: 14px;
+		}
+
+		//
+		// Menu item.
+		//
+
+		.menu-item {
+			position: relative;
+			padding: 0;
+
+			a {
+				display: inline-flex;
+				flex-flow: row nowrap;
+				justify-content: space-between;
+				opacity: 1;
+				color: $white;
+				padding: 10px 20px;
+				max-width: calc( 100% - 50px );
+
+				&:hover,
+				&:focus {
+					opacity: 0.8;
+				}
+
+				@media ( query( min, tablet ) ) {
+					display: flex;
+					max-width: none;
+					padding: 10px;
+					opacity: 0.8;
+
+					&:hover,
+					&:focus {
+						opacity: 1;
+					}
+				}
+			}
+
+			&-has-children {
+				@media ( query( min, tablet ) ) {
+					padding-right: 16px;
+				}
+
+				&::after {
+					content: '▾';
+					position: absolute;
+					top: 0;
+					right: 0;
+					width: 50px;
+					height: 50px;
+					display: flex;
+					justify-content: center;
+					align-items: center;
+					transition: transform 0.2s ease-out;
+					transform-origin: 50% 53%;
+
+					@media ( query( min, tablet ) ) {
+						width: 20px;
+						height: 36px;
+						right: 4px;
+						opacity: 0.8;
+					}
+				}
+			}
+
+			&:hover,
+			&:focus {
```

Comment:
This seems to be the only way to toggle submenu items on mobile with the default WordPress Menu Walker. Since it uses the `:focus` state, the toggle doesn't work when clicking on the icon to close the menu item.

Ideally we would add a button to toggle the state which would require a custom menu walker.

I believe this is quite minor and isn't a big priority right now. @westonruter thoughts?

Reply:
Yeah, that sounds right. Toggling submenus with `:hover`/`:focus` are pretty standard I think.

Reply:
And toggling them off I think it is standard to just click outside.